docs/src/main/tut/plots.md

```diff
@@ -73,79 +73,101 @@ EvilPlot takes an unusual approach to overlaying, in the spirit of building more
 ones. When you want to make a multilayered plot, just make a plot for each layer. We then give you a combinator,
 `Overlay`, to compose all the plots into a meaningful composite.
 
-Let's look at two equally meaningful plots. The first is a
-[Ramachandran plot](https://en.wikipedia.org/wiki/Ramachandran_plot), which is a plot of the phi and psi dihredral
-angles in a peptide bond. This data came from a molecular dynamics simulation of alanine dipeptide, so we have a
-computed "phi" and "psi" from the simulation for each iteration. We want to look at how these two angles were
-distributed, so let's make a contour plot out of the density.
+Let's take a look at some real world data. In its natural state, real world data can be hard to interpret and even harder
+to put to use. Take a look at the public data from https://data.boston.gov/dataset/wicked-free-wi-fi-locations
+containing the Latitude(y) and Longitude(x) coordinates of several free wifi hotspots in Boston. To start, lets parse this
+file into a `Seq[Points]`:
+
+```scala
+import com.cibo.evilplot.numeric.Point
+import scala.collection.mutable.ArrayBuffer
+import scala.io.Source
+import scala.util.Try
+
+val path: String = //Path to saved CSV file
+val data: Seq[Point] = {
```

Comment:
probably cleaner to do it without the mutable collection, e.g.
```scala
val points = source.getLines.tail.map { line =>
	val columns = line.split.map(_.trim)
	Point(columns.head.toDouble, columns(1).toDouble)
}
```
the `scala.util.Try` import looks unnecessary here

Reply:
This works but it won't let me close the bufferedSource for some reason. Is it ok coding practice to just not close it?